test/runtests.jl

```diff
@@ -31,35 +48,60 @@ using Test, OpenEXR, FileIO, Colors, FixedPointNumbers
                 loaded_img = OpenEXR.load(fn)
                 @test typeof(loaded_img) === Array{load_type,2}
                 converted_img = (c -> convert(save_type, c)).(loaded_img)
-                #@test converted_img ≈ loaded_img
+                @test converted_img == img
             finally
                 rm(fn.filename)
             end
         end
     end
 
-    # @testset "Lossy with type conversion" begin
+    @testset "Lossy with type conversion" begin
 
-    #     @testset "Bit depth truncation" begin
+        @testset "Bit depth truncation" begin
+            for (save_type, load_type) in (
+                (RGBA{Float32}, RGBA{Float16}),
+                (RGB{Float32}, RGB{Float16}),
+                (GrayA{Float32}, GrayA{Float16}),
+                (Gray{Float32}, Gray{Float16}),
```

Comment:
Are `Gray{N0f8}`/`RGB{N0f8}` supported here?

Reply:
`N0f8`-based colors are already tested in a section not present in the diff.